application/database/migrations/2023_05_02_133231_add_phone_to_institution_users_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('institution_users', function (Blueprint $table) {
+            $table->string('phone')->nullable();
+        });
+    }
```

Comment:
**[blocker]** While doing another task, I heard from Birgit that is phone numbers are required for users. With that in mind, shouldn’t the column be non-nullable?